src/pages/Admin/Details/Details.tsx

```diff
@@ -1,25 +1,22 @@
 import React from "react";
 
-import * as S from "./Details.styles";
-// Typings
 import { DetailsProps } from "./Details.types";
-// Components
 import DetailsForm from "./Form";
 import DetailsPreview from "./Preview";
 
-// Styles
```

Comment:
Removed import comments as to not trip over the automated import sorting algorithm.